### ISSUE TITLE
Command "project" not found

### ISSUE THREAD

**Issue by geirman** [serverless/serverless-graphql-blog#24]
`$ serverless project install serverless-graphql-blog`

```
  Serverless Error ---------------------------------------

     Command "project" not found, Run "serverless help" for
     a list of all available commands.

  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Forums:        forum.serverless.com
     Chat:          gitter.im/serverless/serverless

  Your Environment Information -----------------------------
     OS:                 darwin
     Node Version:       6.10.2
     Serverless Version: 1.11.0
```

I also tried...
1. `$ serverless install project serverless-graphql-blog`
2. `$ serverless install serverless-graphql-blog`

**Comment by bradennapier** [comment]
Only way I got headway was to use


```
serverless install --url https://github.com/serverless/serverless-graphql-blog
```

It installed things but haven't gotten it going yet.

**Comment by felzix** [comment]
This was written for a serverless version below 1.0. I'm on 1.16.0. So this needs to be updated to be usable.

It would be good if the owner removed this from google searches (or alltogether) so people don't get misdirected here.

**Comment by sbussard** [comment]
@kevinold what would it take to get this updated?

**Comment by sbussard** [comment]
@kevinold what would it take to update this project? it doesn't seem to work anymore

**Comment by serverlesspolska** [comment]
Yes, it would be awesome to have this project updated to recent version of serverless framework (and other libraries as well)

**Comment by bradennapier** [comment]
I do have an example of using this on aws-lambda that is based on what we are using in production now.  It also shows the use of data-loader so I thought some of you guys may find it useful, it's working quite well.  

https://github.com/Dash-OS/graphql-dataloader-example  we combine it with this library to make a clean and simple overall endpoint https://github.com/Dash-OS/aws-lambda-runner

**Comment by Gribbs** [comment]
Please can someone update? Seems like it would do exactly what I need if it would just work on the latest version of serverless!

**Comment by sbussard** [comment]
@Gribbs there's an alternative ... http://apex.run

https://github.com/sbussard/lambda-graphql-apex
